lib/web_ui/test/canvaskit/image_golden_test.dart

```diff
@@ -260,16 +260,33 @@ void _testForImageCodecs({required bool useBrowserImageDecoder}) {
         );
 
         final ui.Image image = (await codec.getNextFrame()).image;
-        // TODO(yjbanov): https://github.com/flutter/flutter/issues/34075
-        // expect(image.width, targetWidth);
-        // expect(image.height, targetHeight);
+        expect(image.width, targetWidth);
+        expect(image.height, targetHeight);
         image.dispose();
         codec.dispose();
       }
 
       testCollector.collectNow();
     });
 
+    test('instantiateImageCodec with multi-frame image does not support targetWidth/targetHeight',
+        () async {
+        final ui.Codec codec = await ui.instantiateImageCodec(
```

Comment:
override `printWarning` like this test to verify that the warning is printed: https://github.com/flutter/engine/blob/7577fd46d50f9bf9827d56fd1ebe75645f1fe53e/lib/web_ui/test/canvaskit/skia_font_collection_test.dart#L24